react/components/Arrow.tsx

```diff
@@ -40,8 +40,13 @@ const Arrow: FC<Props> = ({
   useKeyboardArrows(goBack, goForward)
 
   function handleArrowClick(
-    _: React.MouseEvent<HTMLButtonElement, MouseEvent>
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ) {
+    if (event) {
+      event.stopPropagation();
+      event.preventDefault();
+    }
+
```

Comment:
Can you run a `yarn install` in the root of the project and commit again? It should remove the lint inconsistencies.

Reply:
Done, thank you 👌

Reply:
@kaisermann would you mind taking another look when you've got the time? Thanks a lot 🚀